open_samus_returns_rando/samus_returns_patcher.py

```diff
@@ -48,9 +48,15 @@ def create_custom_init(configuration: dict) -> str:
         atanks = inventory.pop("ITEM_AEION_TANKS")
         max_aeion += atanks * aeion_per_tank
 
+    # These fields are required to start the game
     inventory_update = {
         "ITEM_MAX_LIFE": max_life,
         "ITEM_MAX_SPECIAL_ENERGY": max_aeion,
+        "ITEM_WEAPON_MISSILE_MAX": 0,
+        "ITEM_WEAPON_SUPER_MISSILE_MAX": 0,
+        "ITEM_WEAPON_POWER_BOMB_MAX": 0,
+        "ITEM_METROID_COUNT": 0,
+        "ITEM_METROID_TOTAL_COUNT": 40,
     }
     inventory.update(inventory_update)
```

Comment:
This will not work. If your inventory already contains e.g. `"ITEM_WEAPON_MISSILE_MAX"`, you will overwrite it with 0. You disable starting with any of these items by that.

Reply:
How does Dread bypass this?

Reply:
By doing it the other way around. You update the "must-have" with the starting inventory from the patcher. That way you don't overwrite the `"ITEM_WEAPON_MISSILE_MAX": 0` (so keep it at 0) or it get's overwritten with the value from the patcher.json inventory.
I think, it was at first this way? Afaik I changed it because it wouldn't allow me to define the `ITEM_MAX_LIFE`...but I'm fine with reverting it. So, your starting max life is defined by energy_per_tank + the amount of tanks you start with.

Reply:
Revert as in, go back to what we had before the other pr?

Reply:
Whatever is easier for you.